pbxproj/pbxsections/PBXVariantGroup.py

```
@@ -0,0 +1,30 @@
+            u'sourceTree': source_tree
+        })
+
+    def add_variant(self, file_ref):
+        if not isinstance(file_ref, PBXFileReference):
+            return False
+
+        self.children.append(file_ref.get_id())
+        return True
+
+    def remove_variant(self, file_ref):
+        if not isinstance(file_ref, PBXFileReference):
```

Ruff (0.8.2)

`PBXFileReference` may be undefined, or defined from star imports

(F405)